packages/replay/src/index.ts

```diff
@@ -432,6 +437,8 @@ export class Replay implements Integration {
         // Tag all (non replay) events that get sent to Sentry with the current
         // replay ID so that we can reference them later in the UI
         addGlobalEventProcessor(this.handleGlobalEvent);
+        // We need to filter out dropped events
+        this._overwriteRecordDroppedEvent();
```

Comment:
Because this is in the if block, if we start, stop, and then start again, this will be not overwritten.

Reply:
ah, good point! I'll move this up 👍